go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/datumbrain/jcsv
+module FYP/GO_ASSIGNMENT1/jcsv
```

Comment:
Why?
How can we import and use `FYP/GO_ASSIGNMENT1/jcsv`?

Reply:
When I was trying to run Driver. go, the compiler was generating an error as it was searching the jcsv folder in the src folder inside go. Specifying the path in go.mod resolve that error.